vowpalwabbit/parse_example_json.h

```diff
@@ -1594,8 +1594,8 @@ bool parse_line_json(vw* all, char* line, size_t num_chars, v_array<example*>& e
     // let's ask to continue reading data until we find a line with actions provided
     if (interaction.actions.size() == 0)
     {
-      // VW::return_multiple_example(*all, examples);
-      // examples.push_back(&VW::get_unused_example(all));
+      VW::return_multiple_example(*all, examples);
```

Comment:
why was this fix as easy as uncommenting this? or was it commented by mistake to begin with

Reply:
I talked to Jacob about it, he believes it's okay to uncomment here. This is the original PR: https://github.com/VowpalWabbit/vowpal_wabbit/pull/1878